hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -182,6 +182,64 @@ public function upgrade_1431() {
 
     return TRUE;
   }
+  
+  /**
+   * Sets case type category to Vacancy for application type or
+   * to Workflow if otherwise
+   *
+   * @return bool
+   */
+  public function upgrade_1432() {
+    $customFieldId = CRM_Core_BAO_CustomField::getCustomFieldID('Category');
```

Comment:
What happens here if there's more than one custom field with this same label? Wouldn't it be better to use the API and fetch this based on the field name instead?